tests/apollo/test_skvbc_state_transfer.py

```diff
@@ -279,5 +279,74 @@ async def test_state_transfer_rvt_validity_after_pruning(self, bft_network):
             verify_checkpoint_persistency=False,
             assert_state_transfer_not_started=False)
 
-        # Validate that the root values are in sync after the pruning has finished
+        # Validate that the RVT root values are in sync after the pruning has finished
+        await bft_network.wait_for_replicas_rvt_root_values_to_be_in_sync(bft_network.all_replicas())
+
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    async def test_state_transfer_rvt_root_validation_after_adding_blocks(self, bft_network):
+        """
+        The goal of this test is to validate that all replicas have their Range validation trees (RVTs) synchronized
+        after running the consensus multiple times while there are random restarts and prunings.
+
+        1) Start all replicas in a given BFT network
+        2) Loop 6 times:
+            3) Send enough requests to trigger 2 checkpoints
+            4) After the 1st iteration, do pruning at each even iteration (i.e. i % 2 == 0)
+            5) Randomly choose if a replica will be restarted. If so, select a random replica and restart it
+        6) Wait for the RVT root values to be in sync
+        """
+
+        [bft_network.start_replica(i) for i in bft_network.all_replicas()]
```

Comment:
nit: why the square brackets (list created for?..)

Reply:
There was no need to use list comprehension, thanks. The replicas will now be started in a regular for loop.